test/ctia/test_helpers/aggregate.clj

```diff
@@ -37,17 +37,45 @@
   [values]
   (mapcat set values))
 
+(defn l-get-in
+  "like get-in, but match keys in maps embedded in collections.
+  ~~~clojure
+  > (l-get-in {:a [{:b 2} {:b 3}]} [:a :b])
+    (2 3)
+  > (l-get-in {:a [2 3]} [:a])
+    [2 3]
+  > (l-get-in {:a {:b [2 3]}} [:a :b])
+    [2 3]
+  > (l-get-in {:a [{:b 2} {:b 3}]} [:a :b])
+    (2 3)
+  > (l-get-in {:a {:b 2 :c 3}} [:a :b])
+    2
+  > (l-get-in {:a {:d 2 :c 3}} [:a :b])
+    nil
+  > (l-get-in {:a [{:d 2} {:b 3}]} [:a :b])
+    (3)
+  ~~~"
+  [m ks]
+  (reduce (fn [acc k]
+            (cond
+              (map? acc) (get acc k)
+              (coll? acc) (seq (keep #(get % k) acc))
```

Comment:
what's the behavior is there's a `set` somewhere?

Reply:
It will go to the `(coll? acc)` case
``` clojure
> (map? #{1 2 3})
false
> (coll? #{1 2 3})
true
```